pages/learn/develop/dockerfile.md

```diff
@@ -101,6 +101,20 @@ Here are the supported machine names and architectures:
 
 {{> "general/deviceTypeNames"}}
 
+## Multiple Dockerfiles
+
+There are cases when you would need a higher granularity of control when specifying build instructions for different devices and architectures than a Dockerfile template can provide. An example of this would be when different configuration or installation files are required.
+
+When deploying an application, the balena CLI tool looks at all available dockerfiles and builds the appropriate docker image based on the following order:
```

Comment:
I know there is a note below clarifying that the feature works with `git push` as well, but I still think we could be more precise here, because this statement gives the impression that Dockerfile selection is a CLI feature. In the cases of 'git push' and 'balena push', it is not the CLI that looks at the available dockerfiles, it is the balenaCloud build servers. So maybe rephrase to _"When deploying an application, the balenaCloud build servers or the balena CLI tool look at all available dockerfiles ..."_